designs/developer/add-45/sections/pipelines/pipelines.md

```diff
@@ -46,27 +46,27 @@ version: 4.5
 * The default state of the Add pipeline checkbox is OFF
 * An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default
 
-![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_devpreview.png)
+![Add pipeline](img/Add_Flow_Pipelines_ImportFromGit_devpreview.png)
 
 #### No Pipeline Template Exists
 * When no pipeline template is available (which will be determined after the app, service, or source-to-image builder is added), an Info inline alert shall be displayed within the Pipeline section
 * The alert shall say “There are no pipeline templates available for this runtime.”
 
-![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_No_Temp.png)
+![Add pipeline](img/Add_Flow_Pipelines_ImportFromGit_No_Temp.png)
```

Comment:
Doesn't look like this file is included with this PR, I see "Add_Flow_Pipelines_Docker_No_Temp.png" but not one for ImportFromGit

Reply:
Just added it to the folder so it should be there now.

Reply:
great, looks like it works now.  Thanks @lwrigh !